src/views/activity/activity_item_view.js

```diff
@@ -10,6 +11,11 @@ export default BaseView.extend({
     'select': 'sendSelectionDetails'
   },
 
+  render () {
```

Comment:
`afterRender` might be a closer fit with the other files, unless there's a good reason to use `render` and have to include that blah.prototype.apply boilerplate